# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/util-linux-syssiglist.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- util-linux-2.12a/mount/fstab.c.org	2003-07-15 23:14:41.000000000 +0200
+++ util-linux-2.12a/mount/fstab.c	2004-11-07 23:48:43.374652960 +0100
@@ -376,7 +376,7 @@
 /* Ensure that the lock is released if we are interrupted.  */
 static void
 handler (int sig) {
-     die (EX_USER, "%s", sys_siglist[sig]);
+     die (EX_USER, "%s", strsignal(sig));
 }
 
 static void
